src/validator/Validator.js

```diff
@@ -683,6 +683,11 @@ class ValidationChecker {
           if (!setting) {
             return true;
           }
+
+          if (component.type === 'time') {
```

Comment:
I would rather figure out why the time component is not sending the updated value to the validator instead of hacking the validation routine.